src/components/Tabs/Tab.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import StyledTab from "./Tab.styled";
+
+const Tab = ({ onClick, title, name, active, children, disabled }) => {
+    return <StyledTab name={name} active={active} onClick={disabled ? () => { } : onClick} title={title} disabled={disabled}>
```

Comment:
Jeśli masz zapis, w którym funkcja faktycznie nic nie robi to można napisać po prostu: `onClick={disabled ? null : onClick}`
Potwierdzam też, że `a` nie zalicza się do znaczników, które mają "disabled" - tu wprowadziłęm CIę w błąd, ale już z `button` będzie działać :P Wtedy wystarczy `onClick={onClick}`